docs/basics/chat_interface.md

```diff
@@ -0,0 +1,131 @@
+# Chat Interface
+
+This guide explains how to run the interactive Gradio demo for interacting with models.
+
+## Overview
+
+The chat interface provides a web UI where you can interact with a model. When launched with code execution capabilities, the model can use a Python interpreter to solve problems.
+
+## Launching with Docker
+
+The simplest way to run the chat interface is using Docker.
+
+### Prerequisites
+
+- Docker installed on your system
+- NVIDIA GPU with appropriate drivers
+
+### Steps
+
+1. Create a Dockerfile with the following content:
+
+```dockerfile
+FROM igitman/nemo-skills-vllm:0.6.0 as base
+
+WORKDIR /app/
+RUN chown -R 1000:1000 /app
+
+RUN git clone https://github.com/NVIDIA/NeMo-Skills \
+    && cd NeMo-Skills \
+    && pip install --ignore-installed blinker \
+    && pip install -e . \
+    && pip install gradio \
+    # uncomment if plan to use model that executes Python
+    # && pip install -r requirements/code_execution.txt
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+COPY --chown=1000 entrypoint.sh /app/entrypoint.sh
+RUN chmod +x /app/entrypoint.sh
+ENTRYPOINT [ "/app/entrypoint.sh" ]
+```
+
+2. Create an entrypoint.sh script:
+
+```bash
+#!/bin/bash
+
+# start model server with sandbox inside current container
+CUDA_VISIBLE_DEVICES=0,1 ns start_server \
```

Comment:
```suggestion
ns start_server \
```
is that required? If not, best to remove it and just control with server_gpus parameter